lib/MetaCPAN/Document/Release.pm

```diff
@@ -104,6 +104,12 @@ This is an ArrayRef of modules that are included in this release.
 
 =cut
 
+=head2 deprecated
+
+This is a boolean indicating whether the release is marked as deprecated
+
```

Comment:
I would add:

   ... (the main module's ABSTRACT contains "DEPRECATED" or "DEPRECIATED", or the x_deprecated flag is set in metadata)

Reply:
added as suggested